core/src/main/java/com/linecorp/armeria/client/HttpClientBuilder.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2017 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.client;
+
+import static java.util.Objects.requireNonNull;
+
+import java.net.URI;
+import java.util.function.Function;
+
+import com.linecorp.armeria.common.HttpRequest;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.SerializationFormat;
+
+/**
+ * Creates a new HTTP client that connects to the specified {@link URI} using the builder pattern.
+ * Use the factory methods in {@link HttpClient} if you do not have many options to override.
+ *
+ * @see ClientBuilder Please refer to ClientBuilder for how decorators and HTTP headers are configured
```

Comment:
This will generate HTML like this:

    <a href="....ClientBuilder.html">Please refer to ClientBuilder.....</a>

Instead of hyperlinking the entire sentence which doesn't look nice, please do this:

    Please refer to {@link ClientBuilder} for how ....